components/TextFields/BUILD

```diff
@@ -51,6 +51,7 @@ mdc_public_objc_library(
     ],
     deps = [
         "//components/AnimationTiming",
+        "//components/Buttons",
         "//components/Elevation",
         "//components/Palettes",
         "//components/Typography",
```

Comment:
Please add a dependency on MaterialButtons and MaterialRipple to the snapshot test targets below.

Reply:
Done